client/tests/test_client.py

```diff
@@ -16,6 +17,11 @@ class ClientTestCase(TestCase):
     def setUp(self):
         """Create a dummy user for the purpose of testing."""
         self.api_url = settings.BASE_URL + '/clients/'
+        self.username = 'test_user'
+        self.password = '12345'
+        user = User.objects.create(username='test_user')
```

Comment:
use get_or_create
In big projects there is always a chance may be some one else created this user.

Reply:
As a test database, as it is destroyed every time tests are run, I thought there should not be any user in the start, as `def setUp(self)` function is the first thing which is run, but just to be even more safe, I will update it to `get_or_create`.